webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js

```diff
@@ -132,4 +134,24 @@ describe('Test utils', () => {
         });
     });
   });
+  
+  it('Check that all test specs belong to a test suites', () => {
+    const testFolders = [];
+    for (const [, value] of Object.entries(constants.SUITES)) {
+      value.forEach(path => testFolders.push(path.split('/')[1]));
+    }
+    const getDirectories =   (src, callback) => {
+      glob(src, callback);
+    };
+    //get all spec files in tests/e2e/default
+    getDirectories('../../../../../../tests/e2e/default/**/*.wdio-spec.js', (err, res)  => {
```

Comment:
This function doesn't return anything for me, I think the path doesn't get resolved properly. I think we have to resolve it:
```suggestion
    return getDirectories(path.join(__dirname, '../../../../../../tests/e2e/default/**/*.wdio-spec.js'))
```